.Lib9c.Tools/SubCommend/Create.cs

```diff
@@ -18,7 +18,7 @@ public class Create
         public void Genesis(
             [Option('g')] string gameConfigDir,
             [Option('d')] string goldDistributedPath,
-            [Option('a')] int activationKeyCount)
+            [Option('c')] int activationKeyCount)
```

Comment:
Is this change necessary? If so, how about separate commit for this change?

Reply:
Maybe it was my fault. I'll revert it. 😓